tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -390,6 +390,39 @@ void ModularFileSystem::FlushCaches(TransactionToken* token) {
   if (ops_->flush_caches != nullptr) ops_->flush_caches(filesystem_.get());
 }
 
+Status ModularFileSystem::SetConfiguration(const std::string& name,
```

Comment:
```suggestion
Status ModularFileSystem::SetOption(const std::string& name,
```

And keeping `SetConfiguration` and `GetConfiguration` for setting/getting the full set of options.

Reply:
@mihaimaruseac Done.